convert2rhel/toolopts.py

```diff
@@ -384,7 +385,11 @@ def _process_cli_options(self):
             tool_opts.activation_key = None
 
         if parsed_opts.no_rpm_va:
-            tool_opts.no_rpm_va = True
+            if os.getenv("CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK", None):
+                tool_opts.no_rpm_va = True
+            else:
+                message = "For running without rpm -Va the CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK=1 is required."
```

Comment:
```suggestion
                message = ("We need to run the rpm -Va command for a reliable and complete rollback of changes done to"
                           " the system during the pre-conversion analysis. If you accept the risk of an incomplete"
                           " rollback, set CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK=1. Otherwise, remove the
                           " --no-rpm-va option.")
```